source/controllers/handlePR.js

```diff
@@ -1,59 +1,61 @@
-import config from '../config'
-import GitHub from '../lib/GitHub'
-import Staticman from '../lib/Staticman'
+import config from '../config';
+import GitHub from '../lib/GitHub';
+import Staticman from '../lib/Staticman';
 
 export default async (repo, data) => {
   const ua = config.get('analytics.uaTrackingId')
     ? require('universal-analytics')(config.get('analytics.uaTrackingId'))
-    : null
+    : null;
 
   if (!data.number) {
-    return
+    return;
   }
 
   const github = await new GitHub({
     username: data.repository.owner.login,
     repository: data.repository.name,
-    version: '1'
-  })
+    version: '1',
+  });
 
   try {
-    const review = await github.getReview(data.number)
+    const review = await github.getReview(data.number);
     if (review.sourceBranch.indexOf('staticman_')) {
-      return null
+      return;
     }
 
     if (review.state !== 'merged' && review.state !== 'closed') {
-      return null
+      return;
     }
 
     if (review.state === 'merged') {
-      const bodyMatch = review.body.match(/(?:.*?)<!--staticman_notification:(.+?)-->(?:.*?)/i)
+      const bodyMatch = review.body.match(/(?:.*?)<!--staticman_notification:(.+?)-->(?:.*?)/i);
 
-      if (bodyMatch && (bodyMatch.length === 2)) {
+      if (bodyMatch?.length === 2) {
         try {
-          const parsedBody = JSON.parse(bodyMatch[1])
-          const staticman = await new Staticman(parsedBody.parameters)
+          const parsedBody = JSON.parse(bodyMatch[1]);
+          const staticman = await new Staticman(parsedBody.parameters);
 
-          staticman.setConfigPath(parsedBody.configPath)
-          staticman.processMerge(parsedBody.fields, parsedBody.options)
+          staticman.setConfigPath(parsedBody.configPath);
+          staticman.processMerge(parsedBody.fields, parsedBody.options);
         } catch (err) {
-          return Promise.reject(err)
```

Comment:
Returning here does nothing. This is run as a callback from the webhook library which will just ignore it.